### ISSUE TITLE
Switch from `iron` to a maintained framework

### ISSUE THREAD

**Issue by rnestler** [spaceapi-community/spaceapi-server-rs#109]
The `iron` crate doesn't seem to be maintained anymore. It got last updated 2 years ago (https://crates.io/crates/iron) and depends on hyper 0.10 which is very outdated and causes `cargo audit` to fail.

**Comment by g5pw** [comment]
It looks like `Actix-web` could be a good alternative?

**Comment by dbrgn** [comment]
I've had mixed experience with actix-web.

One option would be to stay low-level and use hyper directly. It's a bit verbose, but not that hard.

Another option would be https://github.com/seanmonstar/warp, but with 43 open PRs I'm not sure how well maintained it is.

I think I'd explore using hyper directly as a server.

**Comment by rnestler** [comment]
> I think I'd explore using hyper directly as a server.

Yes, I was also thinking about that. Using hyper directly as a client wasn't that hard, so I assume for a low complexity thing like we have here it also won't be that hard for a server.
